lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -100,6 +100,7 @@ def script
 set -e
 
 mkdir -p "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+mkdir -p "${PODS_ROOT}/resources-checksums"
```

Comment:
In addition, this would break by cleaning the build products directory, because the build file wouldn't exist anymore but the md5 hash would suggest that.